okhttp/src/main/java/io/grpc/okhttp/OkHttpClientStream.java

```diff
@@ -101,6 +101,11 @@
             transport,
             initialWindowSize,
             method.getFullMethodName());
+
+    // TODO(ejona86) Fix a potential data race using MigratingThreadDeframer in OkHttp.
```

Comment:
You probably need the colon after the TODO. It doesn't matter, but I assume there is some linter somewhere that will complain about that, because somehow that makes us more productive.